.github/actions/setup-rust-runtime/action.yaml

```diff
@@ -34,5 +34,6 @@ runs:
         echo "RUSTC_WRAPPER=sccache" >> $GITHUB_ENV  
         echo "SCCACHE_GHA_ENABLED=true" >> $GITHUB_ENV 
         echo "RUST_BACKTRACE=1" >> $GITHUB_ENV
+        echo "CARGO_INCREMENTAL=false" >> $GITHUB_ENV
```

Comment:
It seems like this may be redundant with the line below (or if the line below isn't working. perhaps we could remove it 🤔 )

Reply:
Probably. I wasn't sure at one point so I put it there to test.